libnymea-core/integrations/thingmanagerimplementation.cpp

```diff
@@ -2274,6 +2397,62 @@ QVariant ThingManagerImplementation::mapValue(const QVariant &value, const State
     return toValue;
 }
 
+void ThingManagerImplementation::registerStateLogger(Thing *thing, const StateTypeId &stateTypeId)
+{
+    StateType stateType = thing->thingClass().getStateType(stateTypeId);
+    QString name = thing->id().toString() + "-" + stateType.name();
+    QList<QVariant::Type> sampledTypes {
+        QVariant::Int,
+        QVariant::UInt,
+        QVariant::LongLong,
+        QVariant::ULongLong,
+        QVariant::Double,
```

Comment:
Floats?

Reply:
Apparently there is no QVariant::Float...